packages/hadron-ipc/src/main.spec.ts

```diff
@@ -0,0 +1,16 @@
+import { expect } from 'chai';
+import ipcMain from './main';
+
+describe('ipcMain', () => {
+  it('should have a respondTo method', () => {
+    expect(ipcMain).to.have.property('respondTo');
+  });
+
+  it('should have a broadcast method', () => {
+    expect(ipcMain).to.have.property('broadcast');
+  });
+
+  it('has a broadcastFocused method', () => {
+    expect(ipcMain).to.have.property('broadcastFocused');
+  });
+});
```

Comment:
Probably not needed now, we could add a test for each environment to make sure this doesn't regress in future. I expect the errors there would break a lot of other tests though so probably not needed.

Reply:
Yeah, this package had surprisingly little amount of tests, I'll add some when I'll move the ipc expose / ipc invoke here, didn't want to do it right away, as this would require some additional changes to the code to make it testable, I wanted to keep the implementation as close to the original as possible for the initial convertion